core/arch/arm/plat-ls/plat_init.S

```diff
@@ -69,23 +69,20 @@ UNWIND(	.fnstart)
 	 *
 	 * SCTLR = 0x00000000
 	 *
-	 * ACTRL = 0x00000041
-	 * - core always in full SMP (FW bit0=1)
+	 * ACTRL = 0x00000040
+	 * - core NOT booted in full SMP (FW bit0=0)
```

Comment:
No, i did not. Actually I don't have a ls1021a.
The fact is that the implementation (line 82) sets ACTRL to 0x40, not 0x41 so I change the comment so that it reflects what the implementation does, rather than changing the initialization value.